### PR TITLE
Update tidb-specific-system-variables.md

### PR DESCRIPTION
**Pull request by bobotu**
### What is changed, added or deleted? (Required)

Update tidb-specific-system-variables.md to include missing variables.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: tidb-specific-system-variables.md [hunk 1]
```diff
@@ -736,3 +784,191 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+
+### tidb_window_concurrency <span class="version-mark">从 v4.0 版本开始引入</span>
+
+作用域：SESSION | GLOBAL
+
+默认值: 4
+
+这个变量用于设置 window 算子的并行度。
+
+### tidb_enable_radix_join
```
Comment (coocood) [inline tidb-specific-system-variables.md:912]: This variable is obsolete, can be removed.

**Comment by coocood** [comment]
LGTM

File: tidb-specific-system-variables.md [hunk 2]
```diff
@@ -484,6 +484,22 @@ mysql> desc select count(distinct a) from test.t;
 
 需要注意的是修改该变量不会对已保存的老数据产生影响，只会对修改变量后的新写入数据使用对应版本格式保存。
 
+### tidb_enable_slow_log
+
+作用域：SESSION
+
+默认值：1
+
+这个变量用于控制是否开启 slow log 功能。
```
Comment (ran-huang) [inline tidb-specific-system-variables.md:493]: Please explain if the default value means `enabled`.

File: tidb-specific-system-variables.md [hunk 3]
```diff
@@ -736,3 +784,183 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+
+这个变量用来控制是否在 profile 输出中标记出对应的 SQL 语句，用于定位和排查性能问题。
+
+### tidb_skip_isolation_level_check
+
+作用域：SESSION
+
+默认值：0
+
+开启这个开关之后对 tx_isolation 赋值一个 TiDB 不支持的隔离级别不会报错。
```
Comment (ran-huang) [inline tidb-specific-system-variables.md:810]:
```suggestion
开启这个开关之后，如果对 `tx_isolation` 赋值一个 TiDB 不支持的隔离级别，不会报错。
```

File: tidb-specific-system-variables.md [hunk 4]
```diff
@@ -736,3 +784,183 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+
+### tidb_low_resolution_tso
+
+作用域：SESSION
+
+默认值：0
+
+这个变量用来设置是否启用低精度 tso 特性，开启该功能之后新事务会使用一个每 2s 更新的 ts 来读取数据。
+主要场景是在可以容忍读到旧数据的情况下，降低小的只读事务获取 tso 的开销。
```
Comment (ran-huang) [inline tidb-specific-system-variables.md:819]: If you want to start a new paragraph, leave a blank line before it.
```suggestion

主要场景是在可以容忍读到旧数据的情况下，降低小的只读事务获取 tso 的开销。
```

File: tidb-specific-system-variables.md [hunk 5]
```diff
@@ -736,3 +784,183 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+
+### tidb_replica_read <span class="version-mark">从 v4.0 版本开始引入</span>
+
+作用域：SESSION
+
+默认值: leader
+
+这个变量用于控制 TiDB 读取数据的位置，有以下三个选择
```
Comment (ran-huang) [inline tidb-specific-system-variables.md:827]:
```suggestion
这个变量用于控制 TiDB 读取数据的位置，有以下三个选择：
```

File: tidb-specific-system-variables.md [hunk 6]
```diff
@@ -736,3 +784,183 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+* follower：只从 follower 节点读取
+* leader-and-follower：从 leader 或 follower 节点读取
+
+### tidb_use_plan_baselines <span class="version-mark">从 v4.0 版本开始引入</span>
+
+作用域：SESSION | GLOBAL
+
+默认值: on
+
+这个变量用于控制是否开启执行计划绑定功能。关于执行计划绑定功能的使用可以参考[执行计划绑定文档](/execution-plan-binding.md#创建绑定)
```
Comment (ran-huang) [inline tidb-specific-system-variables.md:839]: Please specify which value is `enabled` and which value is `disenabled`.

File: tidb-specific-system-variables.md [hunk 7]
```diff
@@ -736,3 +784,183 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+
+### tidb_capture_plan_baselines <span class="version-mark">从 v4.0 版本开始引入</span>
+
+作用域：SESSION | GLOBAL
+
+默认值: off
+
+这个变量用于控制是否开启打开自动创建绑定功能。该功能依赖 statement summary，因此在使用自动绑定之前需打开 Statement Summary 开关。
+开启该功能后会定期遍历一次 statement summary 中的历史 SQL 语句，并为至少出现两次的 SQL 语句自动创建绑定。
```
Comment (ran-huang) [inline tidb-specific-system-variables.md:848]:
```suggestion
这个变量用于控制是否开启自动创建绑定功能。该功能依赖 Statement Summary，因此在使用自动绑定之前需打开 Statement Summary 开关。

开启该功能后会定期遍历一次 Statement Summary 中的历史 SQL 语句，并为至少出现两次的 SQL 语句自动创建绑定。
```

File: tidb-specific-system-variables.md [hunk 8]
```diff
@@ -736,3 +784,183 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+
+### tidb_evolve_plan_baselines <span class="version-mark">从 v4.0 版本开始引入</span>
+
+作用域：SESSION | GLOBAL
+
+默认值: off
+
+这个变量用于控制是否启用自动演进绑定功能。该功能的详细介绍和使用方法可以参考[自动演进绑定文档](/execution-plan-binding.md#自动演进绑定)
+
+为了减少自动演进对集群的影响，可以通过 tidb_evolve_plan_task_max_time 来限制每个执行计划运行的最长时间，其默认值为 600s；通过 tidb_evolve_plan_task_start_time 和 tidb_evolve_plan_task_end_time 可以限制运行演进任务的时间窗口，默认值分别为 00:00 +0000 和 23:59 +0000。
```
Comment (ran-huang) [inline tidb-specific-system-variables.md:858]:
```suggestion
这个变量用于控制是否启用自动演进绑定功能。该功能的详细介绍和使用方法可以参考[自动演进绑定](/execution-plan-binding.md#自动演进绑定)。

为了减少自动演进对集群的影响，可以通过 `tidb_evolve_plan_task_max_time` 来限制每个执行计划运行的最长时间，其默认值为 600s；通过 `tidb_evolve_plan_task_start_time` 和 `tidb_evolve_plan_task_end_time` 可以限制运行演进任务的时间窗口，默认值分别为 `00:00 +0000` 和 `23:59 +0000`。
```

File: tidb-specific-system-variables.md [hunk 9]
```diff
@@ -736,3 +784,183 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+
+这个变量用于控制是否开启向量化执行。
+
+### tidb_enable_index_merge <span class="version-mark">从 v4.0 版本开始引入</span>
+
+作用域：SESSION | GLOBAL
+
+默认值: 0
+
+这个变量用于控制是否开启 index merge 功能
```
Comment (ran-huang) [inline tidb-specific-system-variables.md:926]:
```suggestion
这个变量用于控制是否开启 index merge 功能。
```

File: tidb-specific-system-variables.md [hunk 10]
```diff
@@ -484,6 +484,22 @@ mysql> desc select count(distinct a) from test.t;
 
 需要注意的是修改该变量不会对已保存的老数据产生影响，只会对修改变量后的新写入数据使用对应版本格式保存。
 
+### tidb_enable_slow_log
```
Comment (kissmydb) [inline tidb-specific-system-variables.md:487]: 这篇文档中 variables 和 在数据库中 show 出来的数量不一致，少了 10 几个，比如 'tidb_opt_memory_factor' 没有

Comment (kissmydb) [reply tidb-specific-system-variables.md:487]: 所有的 variables 只提供了 default value，是否能提供一个 Valid Values 的枚举范围，或者 Min Value / Max Value

Comment (bobotu) [reply tidb-specific-system-variables.md:487]: `tidb_batch_commit`, `tidb_batch_delete`, `tidb_batch_insert` 和 `tidb_dml_batch_size` 是被弃用的特殊功能，是有意不写在这里是不希望客户主动去调。

`tidb_enable_radix_join` 之前[提过](https://github.com/pingcap/docs-cn/pull/3159#discussion_r427765313)是弃用功能，所以没写在这里。

`tidb_optimizer_selectivity_level` 和 `tidb_opt_join_reorder_threshold` 这两个也是不希望用户自己去调整的。

像 `tidb_opt_xxxx_factor` 这一族是优化器内部调的参数，之后可能专门做调优相关的文档更新的时候再补。

File: tidb-specific-system-variables.md [hunk 11]
```diff
@@ -736,3 +767,185 @@ TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在
...
+
+### tidb_evolve_plan_task_end_time <span class="version-mark">从 v4.0 版本开始引入</span>
+
+作用域：GLOBAL
+
+默认值：23:59 +0000
+
+这个变量用来设置一天中允许自动演进的结束时间。
+
+### tidb_allow_batch_cop <span class="version-mark">从 v4.0 版本开始引入</span>
```
Comment (yikeke) [inline tidb-specific-system-variables.md:869]: Note that this item has been aligned in https://github.com/pingcap/docs/pull/3002/files#diff-36f1f55efb0294795b77879ecad344a0R575, FYI @CaitinChen

**Pull request merged**